src/components/networked.js

```diff
@@ -319,18 +320,32 @@ AFRAME.registerComponent('networked', {
               childEl.setAttribute(schema.component, schema.property, data);
             }
             else {
-              this.updateComponent(childEl, schema.component, data);
+              var shouldLerp = this.shouldLerpComponent(syncedComponents, schema);
```

Comment:
I am confused, can't you just check `schema.lerp` here?

Reply:
The return value of `NAF.utils.keyToChildSchema` isn't actually a schema. So you cant pull `schema.lerp` off of it. I'd like to rename schema to something else in this context but I'm not really sure what to call it.